app/models/user.rb

```diff
@@ -47,7 +45,7 @@
 #
 class User < ApplicationRecord
   include PgSearch::Model
-
+  self.ignored_columns = ["encrypted_access_token", "encrypted_access_token_iv"]
```

Comment:
This column is not used, im planning on running a migration to remove it in an upcoming commit.